.github/workflows/ci.yml

```
@@ -48,7 +48,7 @@ jobs:
   # Linux builds (containerized builds on GitHub-hosted runners)
   build-linux-debug-gcc-x86_64:
     needs: [filter]
-    if: needs.filter.outputs.should-run == 'true'
+    if: false  # temporarily disabled for AVX-disable CI experiment (PR #11105)
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)